### ISSUE TITLE
Can I clear my gallery?

### ISSUE THREAD

**Issue by tbeseda** [cbh123/charl-e#6]
## Summary

Is it possible to clear my gallery?
After removing the generated images from disk, my gallery shows the prompt strings still. I believe these entries are stored locally and I'm unsure how to remove orphaned prompts.

Thanks!

**Comment by hiddefolkertsma** [comment]
They're stored in `/Users/<username>/Library/Application Support/charl-e/config.json`, you can remove them there. It'd be nice to be able to do that from the interface though 🙂

**Comment by cbh123** [comment]
yes @hiddefolkertsma is exactly right — you could remove them locally. But I'm planning on making this doable in the UI
